### PR TITLE
Add linters to scenario_selector

### PR DESCRIPTION
**Pull request by esteve**

### CODE REVIEW

File: planning/scenario_planning/scenario_selector/package.xml [hunk 1]
```diff
@@ -20,6 +20,9 @@
   <exec_depend>topic_tools</exec_depend>
   <exec_depend>ros2cli</exec_depend>
 
+  <test_depend>ament_lint_auto</test_depend>
+  <test_depend>ament_lint_common</test_depend>
```
Comment (jilaada) [inline planning/scenario_planning/scenario_selector/package.xml:24]: Like in #154 we could change this to just include `ament_cmake_cppcheck`

**Review by jilaada — CHANGES_REQUESTED** [review]
It seems like `CMakeLists.txt` is also missing the linter tests here too.

File: planning/scenario_planning/scenario_selector/package.xml (hunk 1 above)
Comment (esteve) [reply planning/scenario_planning/scenario_selector/package.xml]: Fixed in a8cf7429

**Comment by esteve** [comment]
@jilaada I've added the relevant lines to CMake in 2c38bcca

**Comment by esteve** [comment]
@jilaada I've added the relevant lines to CMake in 2c38bcca

**Comment by esteve** [comment]
@jilaada I've added the relevant lines to CMake in 2c38bcca

**Review by jilaada — APPROVED** [review]
Was able build and run the tests with no errors reported. LGTM :+1:

**Pull request merged**